build.gradle

```diff
@@ -1,20 +1,15 @@
 buildscript {
     repositories {
         mavenLocal()
-        mavenCentral()
+        jcenter()
     }
     dependencies {
-        classpath 'com.mparticle:android-kit-plugin:+'
+        classpath 'com.mparticle:android-kit-plugin:' + project.version
     }
 }
 
 apply plugin: 'com.mparticle.kit'
 
 dependencies {
-    //TODO: Option 1: Replace with your Maven Central Dependency (preferred)
-    //compile 'com.example:example-artifact:1.2.3'
-
-    //TODO: Option 2: Include your jar in ./libs only if you're not in Maven Central
-    //TODO:           Please name the jar based on the version, ie AcmeSDK1234.jar
-    //compile fileTree(dir: 'libs', include: ['*.jar'])
+    compile fileTree(dir: 'libs', include: ['*.jar'])
```

Comment:
Can you audit each of the libs in your libs directory and make sure they're required? in particular, `libs/java-json.jar` I think is a library that mparticle uses for unit testing purposed - if it gets compiled into an app it will break JSON parsing.